applications/acdc/src/acdc_stats_util.erl

```diff
@@ -50,3 +51,14 @@ get_query_limit(JObj) ->
 -spec db_name(ne_binary()) -> ne_binary().
 db_name(Account) ->
     wh_util:format_account_mod_id(Account).
+
+-spec prev_modb(ne_binary()) -> ne_binary().
+prev_modb(Account) ->
+	{{Year, Month, _}, _} = calendar:now_to_universal_time(os:timestamp()),
+	prev_modb(Account, Year, Month-1).
```

Comment:
You could use `{PrevYear, PrevMonth} = kazoo_modb_util:prev_year_month(Year, Month)` so we don't duplicate code. You could also pass in the current MODb and get the previous month's MODb for that account too.